sentry_sdk/tracing.py

```diff
@@ -803,6 +805,36 @@ def finish(self, hub=None, end_timestamp=None):
         pass
 
 
+def trace(func=None):
+    # type: (Any) -> Any
+    """
+    Decorator to start a child span under the existing current transaction.
+    If there is no current transaction, than nothing will be traced.
+
+    Usage:
+        import sentry_sdk
+
+        @sentry_sdk.trace
+        def my_function():
+            ...
+
+        @sentry_sdk.trace
+        async def my_async_function():
+            ...
+    """
+    if PY2:
```

Comment:
style/convention nit but instead of different files can we just switch on the `def` inside `tracing_utils` based on `PY2`, this is the pattern we follow everywhere else.

Reply:
This does not work because it is a `async def`. As soon as the Py2 interpreter sees a `async def` in a file it stops parsing it, thas whey the separate file. 
This is the first case we have where we have a async function in code that is also used in python 2. (the `@trace` decorator should work in py2 and py3. 
All the other cases of async code is code that only works in py3 (because those frameworks only support py3)